packages/neos-ui/src/Containers/Modals/SelectNodeType/nodeTypeFilter.js

```diff
@@ -0,0 +1,34 @@
+import React, {PureComponent} from 'react';
+import PropTypes from 'prop-types';
+import TextInput from '@neos-project/react-ui-components/src/TextInput/';
+import style from './style.css';
+
+class NodeTypeFilter extends PureComponent {
+    static propTypes = {
+        onChange: PropTypes.func.isRequired
+    };
+
+    constructor(props) {
+        super(props);
+
+        this.handleValueChange = this.handleValueChange.bind(this);
+    }
+
+    render() {
+        return (
+            <div className={style.nodeTypeDialogHeader__filter}>
+                <span className={style.nodeTypeFilter__label}>Filter:</span>
```

Comment:
Shall we add a filter icon here?

Reply:
Yeah! Good idea!

Reply:
![bildschirmfoto 2017-09-18 um 10 43 18](https://user-images.githubusercontent.com/769789/30534399-5e417cdc-9c5e-11e7-9c9d-4f1fe2d10a70.png)

Way better!